docs/zk2kraft.html

```diff
@@ -176,5 +176,14 @@ <h5 class="anchor-heading">Removal metrics</h5>
                 Kafka remove all zookeeper dependencies, so the metrics is removed.
             </p>
         </li>
+        <li>
+            <p>
+                Remove the metrics which is monitoring the leader election.
```

Comment:
This seems misleading - I think these metrics are related to the purgatory for the ElectLeader operation - we still have metrics for monitoring leader election.

Reply:
Make sense to me.

Reply:
I think it may be helpful to indicate which metrics replace the ones being removed or if they're not applicable anymore. That would be useful, I believe.

Reply:
@ijuma you are right. #18564 will address it